docs/docsource/Operation-Architecture/Architecture/Cosmos.rst

```diff
@@ -0,0 +1,29 @@
+Cosmos
+======
+
+Cosmos provides translation from CloudServer to the NFS and SMB/CIFS
+file system protocols. It provides a REST API to CloudServer and
+translates CloudServer requests into native NFS or SMB protocol
+requests. Cosmos responds to CloudServer using the S3 protocol. The
+anticipated use case for Cosmos is to enable CloudServer to detect
+changes made to a non-object-storage repository with NFS or SMB/CIFS
+access and to back up these changes to a cloud storage location on a
+regular schedule.
```

Comment:
Just some clarification:
```
Cosmos provides translation from CloudServer to the NFS and SMB/CIFS
file system protocols. It provides a REST API to CloudServer and
translates CloudServer requests into native file system requests.
The anticipated use case for Cosmos is to enable CloudServer to detect
changes made to a non-object-storage repository with NFS or SMB/CIFS
access and to back up these changes to a cloud storage location on a
regular schedule. Cosmos ingests these changes into CloudServer 
using the S3 protocol.
```

Reply:
I have a pedantic beef with Cosmos ingesting something into CloudServer (even if cloudserver called it up). Will this work? 

 "Cosmos serves these changes for CloudServer ingestion over the S3 protocol."

Reply:
Yeah that's totally fine.